src/FsYacc.Core/fsyaccdriver.fs

```diff
@@ -668,6 +678,6 @@ let writeSpecToFile (generatorState: GeneratorState) (spec: ParserSpec) (compile
             generatorState.bufferTypeArgument
             ty
 
-let compileSpec (spec: ParserSpec) (logger: Logger) =
+let compileSpec (spec: ParserSpec) (logger: Logger) quiet =
```

Comment:
Hmm, I would prefer to see `logger` changed to `Microsoft.Extensions.Logging.Logger`.
The `quiet` flag very much feels like a quick win that someone else will need to clean up in the future.